requirements.txt

```diff
@@ -157,7 +157,7 @@ pluggy==1.0.0
     #   tox
 pre-commit==3.2.2
     # via sdv (setup.py)
-protobuf==3.20.3
+protobuf==4.21.12
```

Comment:
also a breaking change, last time I tried we could not generate the stubs from the proto-files and also couldn't use the old ones